src/main/java/com/algorand/algosdk/v2/client/algod/RawTransaction.java

```diff
@@ -0,0 +1,42 @@
+package com.algorand.algosdk.v2.client.algod;
+
+import com.algorand.algosdk.v2.client.common.Client;
+import com.algorand.algosdk.v2.client.common.Query;
+import com.algorand.algosdk.v2.client.common.QueryData;
+import com.algorand.algosdk.v2.client.common.Response;
+
+
+/**
+ * /v2/transactions 
+ */
+public class RawTransaction extends Query {
+
+	public RawTransaction(Client client) {
+		super(client, "post");
+	}
+
+	/**
+	 * The byte encoded signed transaction to broadcast to network 
+	 */
+	public RawTransaction rawtxn(byte[] rawtxn) {
+		addToBody(rawtxn);
+		return this;
+	}
+
+	@Override
+	public Response<String> execute() throws Exception {
```

Comment:
I think this returns a txid, should that be a `Digest`? I'm not sure we need to change this.

Reply:
TxId is defined as String. Until we solve the mystery, it shall be accepted as a glorified String.